### PR TITLE
Publish proto convereter and config-service

### PR DESCRIPTION
**Pull request by pavan-traceable**
## Description
Publish proto-converter and config-service which modules

### CODE REVIEW

**Review by rish691 — DISMISSED** [review]

**Review by JBAhire — APPROVED** [review]
LGTM. Just cut the release once you merge and make sure it’s working. 🙂

**Comment by kotharironak** [comment]
@pavan-traceable you can remove the `Testing`, `Checklist` and `documentation` section from the description as it is just the template part to fill if any.

**Comment by pavan-traceable** [comment]
> @pavan-traceable you can remove the `Testing`, `Checklist` and `documentation` section from the description as it is just the template part to fill if any.

Done

File: config-service/build.gradle.kts [hunk 1]
```diff
@@ -14,6 +14,7 @@ plugins {
   id("org.hypertrace.docker-publish-plugin")
   id("org.hypertrace.integration-test-plugin")
   id("org.hypertrace.jacoco-report-plugin")
+  id("org.hypertrace.publish-plugin")
```
Comment (aaron-steinfeld) [inline config-service/build.gradle.kts:17]: The main config service jar should not be published - let me look what our options are here

Comment (aaron-steinfeld) [reply config-service/build.gradle.kts:17]: Raised a pr into this branch - https://github.com/hypertrace/config-service/pull/27 . It moves the test fixtures to the api package, which is a better home for them.

**Review by aaron-steinfeld — APPROVED** [review]

**Pull request merged**